### PR TITLE
Integrate local devnet

### PR DESCRIPTION
**Pull request by sukantoraymond**
## Why this should be merged
Integrates using local machine as bootstrap validator into subnet deploy command. 

## How this works
// create the local node connected to etna
go run main.go node local start <EtnaInstanceName> --etna-devnet --avalanchego-path <AVALANCHEGO_PATH>

// create a blockchain conf
go run main.go blockchain create <BlockchainName>

// create subnet+blockchain+convert on the blokchain
go run main.go blockchain deploy <BlockchainName> --cluster <EtnaInstanceName> 

## How this was tested
./bin/avalanche subnet deploy will have to achieve L1 set up with local machine as sole bootstrap validator in Etna Devnet

## How is this documented
NA

### CODE REVIEW

File: cmd/contractcmd/init_poa_validator_manager.go [hunk 1]
```diff
@@ -80,6 +80,7 @@ func initPOAManager(_ *cobra.Command, args []string) error {
 			return err
 		}
 	}
+	fmt.Printf("obtained rpc url %s \n", initPOAManagerFlags.rpcEndpoint)
```
Comment (felipemadero) [inline cmd/contractcmd/init_poa_validator_manager.go:83]: remove this

File: cmd/blockchaincmd/deploy.go [hunk 2]
```diff
@@ -762,25 +787,29 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
 			if err != nil {
 				return err
 			}
-			aggregatorExtraPeerEndpoints, err := GetAggregatorExtraPeerEndpoints(network)
+			client, err := evm.GetClient(rpcURL)
+			if err != nil {
+				return err
+			}
+			evm.WaitForChainID(client)
```
Comment (felipemadero) [inline cmd/blockchaincmd/deploy.go:794]: can you make this one to return error and also stop if not healthy here

File: cmd/blockchaincmd/deploy.go [hunk 3]
```diff
@@ -132,6 +135,7 @@ so you can take your locally tested Subnet and deploy it on Fuji or Mainnet.`,
 	cmd.Flags().StringVar(&bootstrapValidatorsJSONFilePath, "bootstrap-filepath", "", "JSON file path that provides details about bootstrap validators, leave Node-ID and BLS values empty if using --generate-node-id=true")
 	cmd.Flags().BoolVar(&generateNodeID, "generate-node-id", false, "whether to create new node id for bootstrap validators (Node-ID and BLS values in bootstrap JSON file will be overridden if --bootstrap-filepath flag is used)")
 	cmd.Flags().StringSliceVar(&bootstrapEndpoints, "bootstrap-endpoints", nil, "take validator node info from the given endpoints")
+	cmd.Flags().BoolVar(&useLocalMachine, "use-local-machine", false, "use local machine as a blockchain validator")
```
Comment (felipemadero) [inline cmd/blockchaincmd/deploy.go:138]: we can make this one true by default

File: cmd/blockchaincmd/deploy.go [hunk 4]
```diff
@@ -466,6 +470,19 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
...
 
+	if !useLocalMachine {
+		ux.Logger.PrintToUser("You can use your local machine as a bootstrap validator on the blockchain")
+		ux.Logger.PrintToUser("This means that you don't have to to set up a remote server on a cloud service (e.g. AWS / GCP) to be a validator on the blockchain.")
+
+		useLocalMachine, err = app.Prompt.CaptureYesNo("Do you want to use your local machine as a bootstrap validator?")
+		if err != nil {
+			return err
+		}
+		if useLocalMachine {
```
Comment (felipemadero) [inline cmd/blockchaincmd/deploy.go:481]: you move this outside of the branch (also needs to improve in future PR)

File: cmd/blockchaincmd/deploy.go (hunk 3 above)
Comment (arturrez) [reply cmd/blockchaincmd/deploy.go:138]: if you pass a clusterName and cluster is Local, it can be used.
No need to add additional flag imho

Comment (felipemadero) [reply cmd/blockchaincmd/deploy.go:138]: ok thats correct

**Review by felipemadero — APPROVED** [review]
LGTM

**Pull request merged**